cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata: 
+  name: monitoring-heapster-v1
+  namespace: default
+  labels: 
+    k8s-app: heapster
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec: 
+  replicas: 1
+  selector: 
+    k8s-app: heapster
+    version: v1
+  template: 
+    metadata: 
+      labels: 
+        k8s-app: heapster
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec: 
+      containers: 
+        - image: gcr.io/google_containers/heapster:v0.13.0
+          name: heapster
+          command:
+            - /heapster
+            - --source=kubernetes:https://kubernetes
+            - --sink=gcl
```

Comment:
This will result in events being pushed to both influxdb and GCL. Shall we update heapster to make events optional in influxdb?